protos/feast/core/FeatureService.proto

```diff
@@ -25,13 +27,19 @@ message FeatureServiceSpec {
 
   // List of features that this feature service encapsulates.
   // Stored as a list of references to other features views and the features from those views.
-  repeated FeatureViewProjection features = 3;
```

Comment:
We should not reuse the id for this - it will cause existing feature services to be unreadable by new versions of feast.

We should remove this field and `reserve` it's id instead for safety.

Reply:
gotcha sg